app/src/main/kotlin/com/wire/android/ui/userprofile/common/UserProfileInfo.kt

```diff
@@ -52,9 +53,9 @@ fun UserProfileInfo(
     ) {
         UserProfileAvatar(
             onClick = onUserProfileClick,
-            isClickable = if (editableState is EditableState.NotEditable) false else !isLoading,
+            isClickable = true,
```

Comment:
Wouldn't this make also other user profile avatars clickable?

Reply:
Ooohh, I made a mistake here too.

It was supposed to remove the `isLoading` part. As the ViewModel has no control about the avatar loading anymore.


Or even leave it the way it was, and just consider if the profile is being loaded.